content/300-guides/200-deployment/100-deployment.mdx

```diff
@@ -165,7 +165,7 @@ The query engine is implemented in Rust and is used by Prisma in the form of exe
 
 Prisma is compatible with AWS RDS Proxy. However, there is no benefit in using it for connection pooling with Prisma due to the way RDS Proxy pins connections:
 
-> "Your connections to the proxy can enter a state known as pinning. When a connection is pinned, each later transaction uses the same underlying database connection until the session ends. Other client connections also can't reuse that database connection until the session ends. The session ends when the client connection is dropped." ~ [AWS RDS Proxy Docs](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/rds-proxy.html)
+> "Your connections to the proxy can enter a state known as pinning. When a connection is pinned, each later transaction uses the same underlying database connection until the session ends. Other client connections also can't reuse that database connection until the session ends. The session ends when the client connection is dropped." - <a href="https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/rds-proxy.html">AWS RDS Proxy Docs</a>
```

Comment:
Did you open an issue about this? Sounds like a serious bug in our link logic that we should fix or it will bite us again in the future.

Reply:
Will open a bug!

Reply:
https://github.com/prisma/docs/issues/1518